packages/playwright-ct-angular/package.json

```diff
@@ -29,35 +29,25 @@
     }
   },
   "dependencies": {
-    "@analogjs/vite-plugin-angular": "0.2.28",
-    "@angular-devkit/build-angular": "^16.1.0",
-    "@playwright/experimental-ct-core": "1.42.0-next",
-    "vite": "^4.4.9"
+    "@playwright/experimental-ct-core": "1.43.0-next"
   },
   "devDependencies": {
-    "@angular/animations": "^16.1.7",
-    "@angular/common": "^16.1.7",
-    "@angular/compiler": "^16.1.7",
-    "@angular/compiler-cli": "^16.1.7",
-    "@angular/core": "^16.1.7",
-    "@angular/platform-browser": "^16.1.7",
-    "@angular/platform-browser-dynamic": "^16.1.7",
-    "@angular/router": "^16.1.7",
-    "@playwright/test": "1.38.1",
+    "@angular/compiler": "^17.0.0",
+    "@angular/core": "^17.0.0",
+    "@angular/platform-browser-dynamic": "^17.0.0",
     "rxjs": "~7.8.1",
-    "tslib": "^2.5.0",
-    "typescript": "^5.0.4",
-    "zone.js": "~0.13.1"
+    "typescript": "~5.2.0",
+    "vite": "~5.0.12",
+    "zone.js": "~0.14.0"
   },
   "peerDependencies": {
-    "@playwright/test": ">=1.38.1",
-    "typescript": ">=4.9.3",
-    "@angular/common": ">=15.1.0 || >=16.0.0",
-    "@angular/platform-browser": ">=15.1.0 || >=16.0.0",
-    "@angular/router": ">=15.1.0 || >=16.0.0",
-    "@angular/core": ">=15.1.0 || >=16.0.0"
+    "@angular/compiler": "^17.0.0",
+    "@angular/core": "^17.0.0",
+    "@angular/platform-browser-dynamic": "^17.0.0",
+    "typescript": ">=5.2.0"
   },
   "bin": {
-    "playwright": "./cli.js"
+    "playwright": "cli.js",
+    "pw-angular": "cli.js"
```

Comment:
`"pw-{framework}": "cli"` was removed in https://github.com/microsoft/playwright/commit/989cf8f179dd3097b5b6cdb5a69153516812c578. I don't think it's unnecessary anymore

Reply:
Done! Thanks